specifyweb/frontend/js_src/lib/components/QueryBuilder/Line.tsx

```diff
@@ -509,17 +510,21 @@ export function QueryLine({
             ))}
           </div>
         ) : (
-          <span className={`${isBasic ? 'col-span-2' : 'contents'}`} />
+          <span
+            className={`
+            ${isBasic ? 'col-span-1' : 'contents'}
```

Comment:
sorry to be so pedantic about this, but could you please maintain consistent indentation?

i.e, the line starting with `$` should be indented
the line **\`}** should be indented too. Like this:

```ts
    className={`
      ${isBasic ? 'col-span-1' : 'contents'}
    `}
```

See that first line and last line has the same indentation, and all others are indented one more.

Or, if there is enough space (to fit everything below 80 chars), you could do it all in one line:

```ts
    className={`${isBasic ? 'col-span-1' : 'contents'}`}
```